exporters/otlp/otlplog/otlploggrpc/example_test.go

```diff
@@ -31,3 +37,64 @@ func Example() {
 	// From here, the provider can be used by instrumentation to collect
 	// telemetry.
 }
+
+func ExampleWithTLSCredentials() {
+	ctx := context.Background()
+	var grpcExpOpt []otlploggrpc.Option
+	// the trusted certificate to use when verifying a server's TLS credentials
```

Comment:
This comment is extremely misleading also given the fact that this is supposed to demonstrate a self-signed certificate.  Self-signed certificates are not inherently trusted by other systems.